packages/nixos/gpu.nix

```diff
@@ -71,6 +71,34 @@ let
           });
         };
       });
+
+  # nix-store-mount-hook mounts the VM's nix store into the container.
+  # TODO(burgerdev): only do that for GPU containers
```

Comment:
Isn't that addressed by placing this in gpu.nix?

Reply:
This comment refers to GPU containers as in containers that have an actual GPU device, not to containers running the GPU image. I'll clarify the comment

Reply:
Are we expecting to start GPU containers without GPU?

Reply:
We use a singular OS image. I don't exactly know what you mean by your question.